spec/presenters/coronavirus/sub_section_json_presenter_spec.rb

```diff
@@ -158,6 +154,142 @@
     end
   end
 
+  describe "#sub_sections" do
+    let(:title) { Faker::Lorem.sentence }
+    let(:label) { Faker::Lorem.sentence }
+    let(:path)  { "/#{File.join(Faker::Lorem.words)}" }
+    let(:sub_section) do
+      create :coronavirus_sub_section, content: "###{title}\n[#{label}](#{path})"
+    end
+
+    it "includes an action link if one exists" do
+      sub_section.action_link_url = "/bananas"
+      sub_section.action_link_content = "Bananas"
+      sub_section.action_link_summary = "Bananas"
+
+      expected = [
+        {
+          list: [
+            {
+              url: "/bananas",
+              label: "Bananas",
+              description: "Bananas",
+              featured_link: true,
+            },
+          ],
+          title: nil,
+        },
+        {
+          list: [
+            {
+              url: path,
+              label: label,
+            },
+          ],
+          title: title,
+        },
+      ]
+
+      expect(subject.sub_sections).to eq(expected)
+    end
+
+    it "does not include an action link if there isn't one" do
+      expected = [
+        {
+          list: [
+            {
+              url: path,
+              label: label,
+            },
+          ],
+          title: title,
+        },
+      ]
+
+      expect(subject.sub_sections).to eq(expected)
+    end
+
+    context "priority_taxon" do
```

Comment:
As this stands this line doesn't capture a context as there's no aspect of "when" to it. It seems to be acting more as a nested describe (which is ok to do).

What I'd suggest is that you have a context here of "when a priority taxon is provided" to test the priority taxon behaviour and then have an inverse of "when a priority taxon is not provided" to confirm it leaves a url unchanged.